src/index.ts

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env node
+
+import { cpSync } from "fs";
+import { join } from "path";
+
+import { Option, type OptionValues, program } from "commander";
+import { select } from "@inquirer/prompts";
+
+type Framework = "angular";
+type TemplateType = "minimal";
+
+interface Options {
+  framework: Framework;
+  template: TemplateType;
+}
+
+program
+  .addOption(
+    new Option("-f, --framework <name>", "front-end framework").choices([
+      "angular",
+    ]),
+  )
+  .addOption(
+    new Option("-t, --template <type>", "template type").choices(["minimal"]),
+  );
+
+program.parse();
+
+main();
+
+async function main() {
+  const opts: Options = await inquire(program.opts());
+  let src: string;
+  let dst: string;
+  switch (opts.framework) {
+    case "angular": {
+      dst = "web3js-angular-dapp-min";
+      src = join(__dirname, "..", "templates", "min", dst);
+    }
+  }
+
+  cpSync(src, join(process.cwd(), dst), { recursive: true });
```

Comment:
I'll suggest to wrap the cpSync() call in a try-catch block to handle/display any errors that may occur during the copying process & specially for displaying feedback message on successful/unsuccessful completion.